.github/workflows/pull-request.yml

```
@@ -168,6 +169,28 @@ jobs:
+    runs-on: ubuntu-latest
+    concurrency:
+      group: pr-${{ github.event.pull_request.number }}
+      cancel-in-progress: true
+    environment:
+      name: preview
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v6
+
+      - name: Setup HTTPS
+        uses: ./.github/actions/setup-https
```

actionlint (1.7.12)

[error] description is required in metadata of "Setup HTTPS" action at "/home/jailuser/git/.github/actions/setup-https/action.yml"

(action)